File: core/src/ru/myitschool/dcrawler/entities/skills/targeting/TargetDisplayer.java
package ru.myitschool.dcrawler.entities.skills.targeting;

import com.badlogic.gdx.utils.Array;
import ru.myitschool.dcrawler.entities.skills.Skill;

/**
 * Created by Voyager on 16.03.2018.
 */
public interface TargetDisplayer {

    boolean targetCreation(int x, int y, Array<TilePos> array, Skill skill);
}

File: core/src/com/wolg_vlad/dcrawler/entities/skills/targeting/TargetDisplayer.kt
package com.wolg_vlad.dcrawler.entities.skills.targeting

import com.wolg_vlad.dcrawler.entities.skills.Skill

/**
 * Created by Voyager on 16.03.2018.
 */
interface TargetDisplayer {
    fun targetCreation(x: Int, y: Int, list: MutableList<TilePos>, skill: Skill): Boolean
}